CMakeLists.txt

```diff
@@ -36,7 +36,11 @@ if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
   )
 endif()
 
-include(${CCF_DIR}/cmake/cpack_settings.cmake)
+if(PACKAGE_TYPE STREQUAL "DEB")
```

Comment:
Do we need this flag? Can we not just set everything, and have cpack use the right vars depending on -G?